### ISSUE TITLE
Uncaught SyntaxError: Invalid regular expression

### ISSUE THREAD

**Issue by Seadragon91** [atom/autoflow#27]
[Enter steps to reproduce below:]

1. Start atom
2. Type in `/***` and then the `character @` (*note:* which because of atom/atom-keymap#35 is actually executing <kbd>Ctrl+Alt+Q</kbd> which executes `autoflow:reflow-selection`)

This problem appears on my PC with Windows 7 64-bit, but this doesn't occur on my linux laptop. I have a german keyboard layout.

**Atom Version**: 0.199.0
**System**: lukas-PC
**Thrown From**: [autoflow](https://github.com/atom/autoflow) package, v0.23.0

### Stack Trace

Uncaught SyntaxError: Invalid regular expression: /^/\*** /: Nothing to repeat

```
At C:\Users\lukas\AppData\Local\atom\app-0.199.0\resources\app.asar\node_modules\autoflow\lib\autoflow.js:42

SyntaxError: Invalid regular expression: /^/\*** /: Nothing to repeat
  at new RegExp (native)
  at C:\Users\lukas\AppData\Local\atom\app-0.199.0\resources\app.asar\node_modules\autoflow\lib\autoflow.js:42:47
  at Object.module.exports.reflow (C:\Users\lukas\AppData\Local\atom\app-0.199.0\resources\app.asar\node_modules\autoflow\lib\autoflow.js:45:13)
  at Object.module.exports.reflowSelection (C:\Users\lukas\AppData\Local\atom\app-0.199.0\resources\app.asar\node_modules\autoflow\lib\autoflow.js:24:27)
  at atom-text-editor.atom.commands.add.autoflow:reflow-selection (C:\Users\lukas\AppData\Local\atom\app-0.199.0\resources\app.asar\node_modules\autoflow\lib\autoflow.js:7:26)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\lukas\AppData\Local\atom\app-0.199.0\resources\app.asar\src\command-registry.js:238:29)
  at C:\Users\lukas\AppData\Local\atom\app-0.199.0\resources\app.asar\src\command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\lukas\AppData\Local\atom\app-0.199.0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:519:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\lukas\AppData\Local\atom\app-0.199.0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:354:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (C:\Users\lukas\AppData\Local\atom\app-0.199.0\resources\app.asar\src\window-event-handler.js:178:20)

```

### Commands

```
  4x -0:23.7.0 editor:newline (atom-text-editor.editor.is-focused)
  2x -0:23.2.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:17.3.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:15.5.0 autoflow:reflow-selection (atom-text-editor.editor.is-focused)
  2x -0:14.3.0 editor:delete-to-beginning-of-word (atom-text-editor.editor.is-focused)
     -0:13.6.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:11.6.0 autoflow:reflow-selection (atom-text-editor.editor.is-focused)
```

### Config

```json
{}
```

### Installed Packages

```coffee
# User
language-powershell, v2.0.1

# Dev
No dev packages
```

**Comment by lee-dohm** [comment]
Are you using a US English keyboard layout? Or some other one?

**Comment by Seadragon91** [comment]
Sorry, a german keyboard layout.

**Comment by Seadragon91** [comment]
Okay, in this case I think the issue can be closed, or? Ty very much, I was confused why that problem only occurs on windows.

**Comment by lee-dohm** [comment]
Well, I think this is still a valid bug ... let me just try to reproduce it.

**Comment by lee-dohm** [comment]
I can reproduce it using Atom v0.200.0-1a8ebd5 on Mac OS X 10.10.3 with the following repro steps:

1. Open Atom
1. Open a new file with <kbd>Cmd+N</kbd>
1. Type `/***` into the editor
1. Execute the `autoflow:reflow-selection` command with <kbd>Cmd+Alt+Q</kbd>

**Expected:** Nothing to happen (no reflow needs to occur)
**Actual:** Error notification appears

**Comment by kevinsawicki** [comment]
Thanks so much for reporting this, it will be fixed in the next Atom release, 0.201.0
